PatchKit/Classes/Views/ImagesStackView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ImagesStackView.swift
+//  abseil
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import UIKit
+
+class ImagesStackView: UIView {
+    
+    private var imagesStackView = UIStackView()
```

Comment:
this is fine but just curious why you chose a stack view over a collection view. using a stack view makes you have to handle the logic of rows by yourself

Reply:
Didn't wanna deal with nested scrolling plus the sizing of the collectionview. Cuz were nesting the whole form, not just the collectionview, in a scrollview, we'd have to force the collectionview to expand to full size which is effectively the same. Can change in a later pr if we feel thats more efficient